apps/server/src/modules/tool/external-tool/service/datasheet-pdf.service.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from '@nestjs/common';
+import { createPdf, TCreatedPdf } from 'pdfmake/build/pdfmake';
+import pdfFonts from 'pdfmake/build/vfs_fonts';
+import { StyleDictionary } from 'pdfmake/interfaces';
+import { ExternalToolDatasheetTemplateData, ExternalToolParameterDatasheetTemplateData } from '../domain';
+
+@Injectable()
+export class DatasheetPdfService {
+	public generatePdf(templateData: ExternalToolDatasheetTemplateData): Promise<Buffer> {
+		const deactivated: string = templateData.isDeactivated ? 'Dieses Tool ist deaktivert.' : '';
```

Comment:
if you use isDeactivated like a boolean than make it one

Reply:
or i'll keep it as a string and use it as one, too :)